HW0/P6/P6B.py

```diff
@@ -16,16 +16,26 @@ def burnTime(t):
 
     # Use a variety of wait times
     ratio = []
-    wait_time = []
+    wait_time = [10e-6,10e-5,10e-4,10e-3,10e-2,10e-1]
 
     for t in wait_time:
         # Compute jobs serially and in parallel
         # Use time.time() to compute the elapsed time for each
         serialTime = 1
         parallelTime = 1
+        start1 = time.time()
```

Comment:
Yes, this is because xrange returns an iterator (small space) rather than a list containing all the elements as range does.